projects/lights_control/src/lights_gpio.c

```diff
@@ -62,6 +63,7 @@ StatusCode lights_gpio_process_event(const LightsGpio *lights_gpio, const Event
   if ((e->id != LIGHTS_EVENT_GPIO_OFF) && (e->id != LIGHTS_EVENT_GPIO_ON)) {
     return STATUS_CODE_OK;
   }
+  LOG_DEBUG("Turning %s: %d\n", (e->id == LIGHTS_EVENT_GPIO_ON) ? "ON" : "OFF", e->data);
```

Comment:
If this is just for debugging it can probably be removed? But up to you as it might help with integration.

Reply:
it's pretty generic. I think it's good to be there for now  as I'm debugging this with hardware and stuff.